apps/meteor/app/ui/client/lib/recorderjs/audioEncoder.js

```diff
@@ -40,7 +40,10 @@ class AudioEncoder extends Emitter {
 	handleWorkerMessage = (event) => {
 		switch (event.data.command) {
 			case 'end': {
-				const blob = new Blob(event.data.buffer, { type: 'audio/mpeg' });
+				// prepend mp3 magic number to the buffer
+				const magicNoPrefix = new Int8Array([73, 68, 51, 3, 0, 0, 0, 0, 0, 0]);
```

Comment:
Just a quick thing: can you check the buffer doesn't have the magic number before preppending? That way if the provider changes and preppends it, we won't have problems 👀

Reply:
I don't think this check is necessary because we'll always have only audio-related data in `event.data.buffer` & not any headers